packages/contracts-rfq/contracts/interfaces/IFastBridgeV2.sol

```diff
@@ -20,4 +20,33 @@ interface IFastBridgeV2 is IFastBridge {
     /// @param request The encoded bridge transaction to claim on origin chain
     function claim(bytes memory request) external;
 
+
+
+
+
+
+
+    enum BridgeStatus {
+        NULL, // doesn't exist yet
+        REQUESTED,
+        RELAYER_PROVED,
+        RELAYER_CLAIMED,
+        REFUNDED
+    }
+
+    struct BridgeTxDetails {
+        BridgeStatus status;
+        BridgeProof proof;
+    }
```

Comment:
This will not have a desired effect of saving the gas on reads and writes, as the internal composition of the struct takes more than a single storage slot (256 bits). We don't export a gas report as of now, but in this case it's possible to generate using `forge t --mc FastBridgeV2SrcTest --nmt revert --gas-report`.
- Using `FastBridgeV2SrcTest` test contract gives us the overview for the source chain-related functions
- `--nmt revert` excludes test functions having a word "revert" in the name, which makes the report more useful (revert cases in general consume less gas, and thus need to be filtered)

Reply:
This are the results when run on master, prior to this PR
![image](https://github.com/user-attachments/assets/1a227476-bf8f-42e1-a4d8-e28e66c8ad40)

Reply:
And this are the results after this PR changes:
![image](https://github.com/user-attachments/assets/6ed04e1f-85c4-49e0-82d3-d4942c073790)

Reply:
To gain the performance in gas consumption, the `BridgeTxDetails` needs to fit within 256 bits:
- `BridgeStatus` enum is equivalent to `uint8`
- `address relayer` takes 160 bits
- which leaves up to 88 bits for the proof's timestamp

This is an opportunity to follow the suit of #3170 and also store the block number, when the prove happened. This could be useful for some of the Relayer implementations. This will require using up to 88 bits for both block number and timestamp, which is more than enough for practical purposes.